packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -42,6 +42,7 @@ const AccountSwitcher = props => {
     const [is_dxtrade_demo_visible, setDxtradeDemoVisible] = React.useState(true);
     const [is_dxtrade_real_visible, setDxtradeRealVisible] = React.useState(true);
     const [exchanged_rate, setExchangedRate] = React.useState('');
+    const [exchanged_rate_mt5_dxtrade, setExchangedRateMT5Dxtrade] = React.useState({ mt5_demo: '', dxtrade: '' });
```

Comment:
From what I understand, the code you added/updated here is related to demo accounts. Perhaps it is then better to change dxtrade to dxtrade_demo then for better readability.

`React.useState({ mt5_demo: '', dxtrade_demo: '' });`